newIDE/app/src/UI/User/UserChip.js

```diff
@@ -58,15 +53,6 @@ const UserChip = ({ onOpenProfile }: Props) => {
   ) : (
     <div style={styles.buttonContainer}>
       <LineStackLayout noMargin alignItems="center">
-        <FlatButton
```

Comment:
should we still display it on desktop?
It prevents an additional click, but it's true that it's not the most common flow if you only have one computer & one account

Reply:
I would avoid it for now, it's not super common, and will avoid a click but I think it's a well known pattern that "Create account will also show me a link to login". A signed up users is also someone who has used GDevelop a few times, so they kind of know it. Finally, in the worst case, they can find it in other part of the home screens.